apps/api/v2/src/ee/calendars/services/apple-calendar.service.ts

```diff
@@ -73,14 +75,35 @@ export class AppleCalendarService implements CredentialSyncCalendarApp {
       invalid: false,
     };
 
+    const existingAppleCalendarCredentials = await this.credentialRepository.getAllUserCredentialsByTypeAndId(
+      APPLE_CALENDAR_TYPE,
+      userId
+    );
+
+    if (existingAppleCalendarCredentials && existingAppleCalendarCredentials.length > 0) {
+      const hasUpdatedCalendar = existingAppleCalendarCredentials.filter((calendarCredential: Credential) => {
+        const decryptedKey = JSON.parse(
+          symmetricDecrypt(calendarCredential.key, process.env.CALENDSO_ENCRYPTION_KEY || "")
```

Comment:
Has TS error:
<img width="1061" alt="Screenshot 2024-09-04 at 09 13 18" src="https://github.com/user-attachments/assets/d8931c4e-6238-4df1-a7c5-425adaee388a">

Reply:
Is this error still persisting after my recent changes?